tokens/style-dictionary.js

```diff
@@ -154,4 +154,24 @@ StyleDictionary.registerFormat({
   },
 });
 
+/**
+ * Formatter to generate CSS custom media queries.
+ * Gets input about existing tokens of the 'size' category,
+ * 'breakpoints' subcategory, and generates a CSS custom media queries.
+ */
+StyleDictionary.registerFormat({
+  name: 'css/custom-media',
+  formatter({ dictionary, file }) {
+    const { size: { breakpoint } } = dictionary.properties;
+
+    let customMediaVariables = '';
+
+    Object.keys(breakpoint).forEach(key => {
```

Comment:
you could use `Object.values` here to make it a little prettier, e.g.
```javascript
Object.values(breakpoint).forEach(item => {
   customMediaVariables += `@custom-media --${item.name} (max-width: ${item.value}); \n`;
});
```

Reply:
Thanks 👍